Tutorial_5_BonusNotebook.ipynb

```diff
@@ -0,0 +1,756 @@
+    "  \n",
+    "\n",
+    "`numthreads=!nproc`  \n",
+    "`numthreadsint = int(numthreads[0])`  \n",
+    "`%env CORES = $numthreadsint`  \n",
+    "`echo ${CORES}`  \n",
+    "    \n",
+    "</details>\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### STEP 3: Downloading relevant FASTQ files using SRA Tools\n",
+    "\n",
+    "As before, we need to download FASTQ data from SRA. We mentioned at the beginning of this tutorial that we are only using 3 out of the 25 time points from the experiment. The Sequence Run Project for this manuscript is [SRP261441](https://www.ncbi.nlm.nih.gov/sra/?term=SRP261441). We're going to select time points 1, 13, and 25. Look through the SRP page and find the accession numbers for those time points. We will use them in the next step."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "    \n",
+    "    Accession numbers are SRR11784387, SRR11784398, and SRR11784410\n",
+    "  \n",
+    "</details>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Add those accession numbers to your accs.txt file and have a look at it to make sure they are correct."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cat accs.txt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Downloading multiple files using the SRA-toolkit.\n",
+    "\n",
+    "Next we are going to download the FASTQ files using the sra-toolkit. We are going to do this in two steps: prefetch followed by fasterq-dump. Write the command below that will prefetch the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "   \n",
+    "    \n",
+    "   `prefetch -O data/raw_fastq/ --option-file accs.txt `\n",
+    "   \n",
+    "  \n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Converting Multiple SRA files to Fastq\n",
+    "\n",
+    "As before, use a loop to convert each file in the list."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "   \n",
+    "\n",
+    "`for x in `cat accs.txt`; do fasterq-dump -f -O data/raw_fastq -e $CORES -m 4G data/raw_fastq/$x/$x.sra; done`  \n",
```

Comment:
Change this help code to, so the user can see the backticks (``):
 <details>
  <summary>Click for help</summary>
<pre>
<code>for x in `cat accs.txt`; do fasterq-dump -f -O data/raw_fastq -e $CORES -m 4G data/raw_fastq/$x/$x.sra; done </code>
</pre>
   
  
</details>

Reply:
Fixed.